content/en/serverless/azure_app_services.md

```diff
@@ -41,9 +42,16 @@ The Datadog extension for Azure App Services provides additional monitoring capa
 
 1. If you haven't already, set up the [Microsoft Azure integration][8] first.
 
-2. The extension only supports Azure App Service Web Apps. **Function Apps are not supported**.
+2. The extension only supports:
 
-<div class="alert alert-warning">To be notified when a private beta is available for Function Apps or additional runtimes, <a href="https://forms.gle/n4nQcxEyLqDBMCDA7">sign up here</a>.</div>
+    - Azure App Service Web Apps
```

Comment:
@ruthnaebeck I made some modifications directly in the file here that I think make it read better. Let me know if you have any feedback or changes you want to make

Reply:
I'm getting a message that my commit failed. Not sure what's going on there, but I think number 2 under Setup>Requirements should read:

2. The extension can be installed on the following resource types:
  - Azure App Service Web Apps
  - **Beta** Function Apps hosted on Premium or Standard App Service Plans

    **Beta Notes**:
      - Function tracing is supported in extension v 1.3.4+
      - Function Apps on consumption plans are not supported.
      - There are no billing implications for tracing Function apps during the beta period.

    <div class="alert alert-warning">Interested in support for other App Service resource types or runtimes? <a href="https://forms.gle/n4nQcxEyLqDBMCDA7">Sign up here</a> to be notified when a beta becomes available.</div>